### PR TITLE
doc: Rename libraries/bluetooth_services to libraries/bluetooth/

### PR DESCRIPTION
**Pull request by umapraseeda**
Renamed libraries/bluetooth_services to libraries/bluetooth/ to make the naming of the doc folder consistent with include/bluetooth

### CODE REVIEW

**Comment by umapraseeda** [comment]
Added https://github.com/nrfconnect/sdk-nrf/pull/17122/commits/0150380929d017eda086a1b8c5c3e45a0f9d599c after discussions with @gmarull

File: doc/_utils/redirects.py [hunk 1]
```diff
@@ -430,6 +430,107 @@
     ("samples/wifi/sr_coex/README", "samples/wifi/ble_coex/README"), # Wi-Fi: Bluetooth LE coexistence
     ("samples/samples_zigbee", "samples/zigbee"), # Zigbee samples (landing)
     ("samples/samples_other", "samples/other"), # Other samples (landing)
+    ("libraries/bluetooth_services/index", "libraries/bluetooth/index"), # Changed the folder name libraries/bluetooth_services to libraries/bluetooth to better match the include folder structure
```
Comment (divipillai) [inline doc/_utils/redirects.py:433]: Also the tag file would require an update - nrf\doc\_zoomin\ncs.tags - line no: 366, 370, 371.

File: doc/_utils/redirects.py [hunk 2]
```diff
@@ -430,6 +430,107 @@
...
+    ("libraries/bluetooth_services/mesh/vnd/dm", "libraries/bluetooth/mesh/vnd/dm"),
+    ("libraries/bluetooth_services/mesh/vnd/dm_srv", "libraries/bluetooth/mesh/vnd/dm_srv"),
+    ("libraries/bluetooth_services/mesh/vnd/dm_cli", "libraries/bluetooth/mesh/vnd/dm_cli"),
+    ("libraries/bluetooth_services/mesh/vnd/le_pair_resp", "libraries/bluetooth/mesh/vnd/le_pair_resp"),
+    ("libraries/bluetooth_services/mesh/models", "libraries/bluetooth/mesh/models"),
+    ("libraries/bluetooth_services/mesh/properties", "libraries/bluetooth/mesh/properties"),
+    ("libraries/bluetooth_services/mesh/dk_prov", "libraries/bluetooth/mesh/dk_prov"),
+    ("libraries/bluetooth_services/mesh/sensor", "libraries/bluetooth/mesh/sensor"),
+    ("libraries/bluetooth_services/mesh/sensor_types", "libraries/bluetooth/mesh/sensor_types"),
+    ("libraries/bluetooth_services/radio_notification_conn_cb", "libraries/bluetooth/radio_notification_conn_cb"),
```
[the remark this answers is not in the document]
Comment (umapraseeda) [reply doc/_utils/redirects.py:508]: Yes, good catch.

File: doc/_zoomin/ncs.tags.yml [hunk 3]
```diff
@@ -363,12 +363,12 @@ mapping_topics:
   - nrf/libraries/bin/lwm2m_carrier/*.html: ["nrf91-series", "nrf9161", "nrf9151", "nrf9160", "lte",
                                              "development-kits"]
   - nrf/libraries/bin/lwm2m_carrier/certification.html: ["certification"]
-  - nrf/libraries/bluetooth_services/*.html: ["ble", "nrf52-series", "nrf52840", "nrf52833",
+  - nrf/libraries/bluetooth/*.html: ["ble", "nrf52-series", "nrf52840", "nrf52833",
                                               "nrf52832", "nrf52820", "nrf52811", "nrf52810",
                                               "nrf52805", "nrf54-series", "nrf54h20", "nrf54l15",
                                               "nrf54l05", "nrf54l10", "development-kits"]
```
Comment (divipillai) [inline doc/_zoomin/ncs.tags.yml:369]:
```suggestion
  - nrf/libraries/bluetooth/*.html: ["ble", "nrf52-series", "nrf52840", "nrf52833", "nrf52832", 
                                     "nrf52820", "nrf52811", "nrf52810", "nrf52805", "nrf54-series", 
                                     "nrf54h20", "nrf54l15", "nrf54l05", "nrf54l10", 
                                     "development-kits"]
```

**Review by divipillai — APPROVED** [review]
Approved with a minor suggestion.

**Review by b-gent — APPROVED** [review]

**Review by richabp — APPROVED** [review]

**Review by gmarull — APPROVED** [review]

**Pull request merged**